macros/edr/system/system_utils/get_config_var.sql

```diff
@@ -16,6 +16,7 @@
     'disable_model_alerts': false,
     'disable_test_alerts': false,
     'disable_run_results': false,
+    'disable_skipped_model_alerts': false,
```

Comment:
Could you please change the default value to be `true`?
We've figured out that it just confuses users more than it helps.